lib/compat/wordpress-6.5/fonts/class-wp-font-utils.php

```diff
@@ -20,11 +20,41 @@
 	 * @access private
 	 */
 	class WP_Font_Utils {
+
+		/**
+		 * Adds surrounding quotes to font family names that contain special characters.
+		 *
+		 * It follows the recommendations from the CSS Fonts Module Level 4.
+		 * @link https://www.w3.org/TR/css-fonts-4/#font-family-prop
+		 *
+		 * @since 6.5.0
+		 * @access private
+		 *
+		 * @see sanitize_font_family()
```

Comment:
`@see` is for referencing other things that this method relies on, so doesn't seem needed here (https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods)